charts/scalardl-audit/templates/_helpers.tpl

```diff
@@ -0,0 +1,74 @@
+{{/* vim: set filetype=mustache: */}}
```

Comment:
```$ diff -u scalardl/templates/_helpers.tpl scalardl-audit/templates/_helpers.tpl
--- scalardl/templates/_helpers.tpl     2021-06-22 17:12:01.000000000 +0900
+++ scalardl-audit/templates/_helpers.tpl       2021-08-17 15:22:03.000000000 +0900
@@ -2,7 +2,7 @@
 {{/*
 Expand the name of the chart.
 */}}
-{{- define "scalardl.name" -}}
+{{- define "scalardl-audit.name" -}}
 {{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
 {{- end }}

@@ -11,7 +11,7 @@
 We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
 If release name contains chart name it will be used as a full name.
 */}}
-{{- define "scalardl.fullname" -}}
+{{- define "scalardl-audit.fullname" -}}
 {{- if .Values.fullnameOverride -}}
 {{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" -}}
 {{- else -}}
@@ -27,16 +27,16 @@
 {{/*
 Create chart name and version as used by the chart label.
 */}}
-{{- define "scalardl.chart" -}}
+{{- define "scalardl-audit.chart" -}}
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" }}
 {{- end }}

 {{/*
 Common labels envoy
 */}}
-{{- define "scalardl-envoy.labels" -}}
-helm.sh/chart: {{ include "scalardl.chart" . }}
-{{ include "scalardl-envoy.selectorLabels" . }}
+{{- define "scalardl-audit-envoy.labels" -}}
+helm.sh/chart: {{ include "scalardl-audit.chart" . }}
+{{ include "scalardl-audit-envoy.selectorLabels" . }}
 {{- if .Chart.AppVersion }}
 app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 {{- end }}
@@ -44,11 +44,11 @@
 {{- end }}

 {{/*
-Common labels ledger
+Common labels auditor
 */}}
-{{- define "scalardl-ledger.labels" -}}
-helm.sh/chart: {{ include "scalardl.chart" . }}
-{{ include "scalardl-ledger.selectorLabels" . }}
+{{- define "scalardl-audit-auditor.labels" -}}
+helm.sh/chart: {{ include "scalardl-audit.chart" . }}
+{{ include "scalardl-audit-auditor.selectorLabels" . }}
 {{- if .Chart.AppVersion }}
 app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 {{- end }}
@@ -58,17 +58,17 @@
 {{/*
 Selector labels envoy
 */}}
-{{- define "scalardl-envoy.selectorLabels" -}}
-app.kubernetes.io/name: {{ include "scalardl.name" . }}
+{{- define "scalardl-audit-envoy.selectorLabels" -}}
+app.kubernetes.io/name: {{ include "scalardl-audit.name" . }}
 app.kubernetes.io/instance: {{ .Release.Name }}
 app.kubernetes.io/app: envoy
 {{- end }}

 {{/*
-Selector labels ledger
+Selector labels auditor
 */}}
-{{- define "scalardl-ledger.selectorLabels" -}}
-app.kubernetes.io/name: {{ include "scalardl.name" . }}
+{{- define "scalardl-audit-auditor.selectorLabels" -}}
+app.kubernetes.io/name: {{ include "scalardl-audit.name" . }}
 app.kubernetes.io/instance: {{ .Release.Name }}
-app.kubernetes.io/app: ledger
+app.kubernetes.io/app: auditor
 {{- end }}
```